float64 throttle
float64 radius
---
bool success